src/screens/FeedScreen/Content.js

```diff
@@ -53,7 +54,6 @@ const Content = ({
   const [amountCut, setAmountCut] = React.useState(0);
   const [textCut, setTextCut] = React.useState(null);
   const [arrText] = React.useState([]);
-  const layoutHeight = dimen.size.FEED_CONTENT_HEIGHT;
   const isIos = Platform.OS === 'ios';
 
   React.useEffect(() => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded layout height has been removed and replaced with a dynamic value set by the `handleHeightContainer` function. This should make the component more flexible and responsive to different content sizes.
```diff
- const layoutHeight = dimen.size.FEED_CONTENT_HEIGHT;
+ const [layoutHeight, setLayoutHeight] = React.useState(null);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->